CMakeLists.txt

```diff
@@ -66,9 +66,6 @@ message(STATUS "Configuring MAM4xx v${MAM4XX_VERSION}")
 # Precision of floating point numbers.
 message(STATUS "Using ${HAERO_PRECISION} precision floating point numbers")
 
-# We build static libraries only.
-set(BUILD_SHARED_LIBS OFF)
```

Comment:
@jeff-cohere, @mahf708. Should we add CMake options to turn BUILD_SHARED_LIBS on and off, or can we set BUILD_SHARED_LIBS=ON/OFF directly from the CMake configuration?